packages/stream-tile/src/tile-document.ts

```diff
@@ -191,7 +191,8 @@ export class TileDocument<T = Record<string, any>> extends Stream {
       // document as there shouldn't be any existing state for this doc on the network.
       opts.syncTimeoutSeconds = 0
     }
-    const commit = await TileDocument.makeGenesis(ceramic, content, metadata)
+    const signer: CeramicSigner = opts.asDID ? { did: opts.asDID } : ceramic
```

Comment:
is this related to the core change in this PR?  Or is this a separate bug fix you noticed while in there?

Reply:
This is needed to add the ability to create a stream with a CACAO enabled DID. This was not supported before.